weave/trace_server/trace_server_interface.py

```diff
@@ -94,6 +94,8 @@ class CallSchema(BaseModel):
 
     # Start time is required
     started_at: datetime.datetime
+    started_at_micros: Optional[datetime.datetime] = None
```

Comment:
i think we can just use started_at and ended_at directly here with higher precision if avail